azure/inputseeders/azure/id.ftl

```diff
@@ -0,0 +1,139 @@
+[#ftl]
+
+[@addInputSeeder
+    id=AZURE_INPUT_SEEDER
+    description="Azure provider inputs"
+/]
+
+[@addSeederToInputStage
+    inputStage=MASTERDATA_SHARED_INPUT_STAGE
+    inputSeeder=AZURE_INPUT_SEEDER
+/]
+
+[@addSeederToInputStage
+    inputStage=MOCK_SHARED_INPUT_STAGE
+    inputSeeder=AZURE_INPUT_SEEDER
+/]
+
+[@addSeederToInputStage
+    inputSources=[MOCK_SHARED_INPUT_SOURCE]
+    inputStage=COMMANDLINEOPTIONS_SHARED_INPUT_STAGE
+    inputSeeder=AZURE_INPUT_SEEDER
+
+/]
+
+[#macro azure_input_loader path]
```

Comment:
Macro naming structure might be better as 

`azure_inputloader`

Reply:
done